br/pkg/aws/ebs.go

```diff
@@ -149,8 +149,18 @@ func (e *EC2Session) CreateSnapshots(backupInfo *config.EBSBasedBRMeta) (map[str
 				resp, err := e.createSnapshotsWithRetry(context.TODO(), &createSnapshotInput)
 
 				if err != nil {
-					return errors.Trace(err)
+					if aErr, ok := err.(awserr.Error); !ok || aErr.Code() != "TagLimitExceeded" {
+						return errors.Trace(err)
+					}
+
+					log.Warn("tag number exceeds AWS limitation during snapshots, retry without tagging")
+					createSnapshotInput.SetTagSpecifications(nil)
```

Comment:
It seems we are going to create a snapshot / volume with no tags. Is such a snapshot / volume still expected?